src/sql/util.py

```diff
@@ -6,6 +6,8 @@
 from sql.store import store, _get_dependents_for_key
 from sql import exceptions, display
 import json
+from pathlib import Path
+import toml
```

Comment:
do not make toml a top-level import because this is an optional dependency, you can do something like this:

```python
try:
     import toml
except ModuleNotFoundError:
    toml = None

and then, wherever you need it, you can use the requires decorator to raise an error if the package is missing and a user tries to call the function: https://ploomber-core.readthedocs.io/en/latest/dependencies.html
```

Reply:
modified `try: except:` block and added `@requires(["toml"])` to `load_toml`